build.sbt

```diff
@@ -21,6 +20,7 @@ val `tasty-core` = Build.`tasty-core`
 val `tasty-core-bootstrapped` = Build.`tasty-core-bootstrapped`
 val `tasty-core-scala2` = Build.`tasty-core-scala2`
 val scaladoc = Build.scaladoc
+val `scaladoc-nonBootstrapped` = Build.`scaladoc-nonBootstrapped`
```

Comment:
Any particular reason to not make the nonbootstrapped the default one like for the other projects? Also if this stays in, this should be renamed to `nonbootstrapped` all lowercase to match `scala3-tasty-inspector-nonbootstrapped` above

Reply:
The *-bootstrapped projects are ones that we want to used in most cases so having a longer name there is a bit misleading. Also both `scaladoc` and `tastyinspector` are currently used in bootstrapped context so I didn't wanted to change all this places to `scaladoc-bootstrapped`